pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-cluster-admin.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-cluster-admin
+rules:
+- apiGroups:
+  - ''
+  - user.openshift.io
+  resources:
+  - groups
+  - users
+  - identities
+  - useridentitymappings
+  verbs:
+  - create
+  - delete
+  - patch
+  - update
+- apiGroups:
+  - ''
+  - authorization.openshift.io
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  - clusterrolebindings
```

Comment:
Need to verify if this is safe.

Reply:
Yes, the other thing apart from priviledge escalation to test is access in the protected namespaces.

Reply:
According to @abhgupta,
```
This is required to allow dedicated admins to assign/remove cluster level roles (good example would be view and self-provisioner roles) to other users. Also, we had noticed early on that the ability to get/list these resources was required to enable other functionality related to user and group management (I think - its been a while... or perhaps to run commands like who-can).
```

For now I think we should remove this until required.